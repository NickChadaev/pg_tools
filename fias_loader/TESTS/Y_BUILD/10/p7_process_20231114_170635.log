2023-11-14 17:06:35 home/n.chadaev@abrr.local
127.0.0.1 5432 unsi_test_10 postgres /home/n.chadaev@abrr.local/Y_BUILD/10/stage_gar_c_10.csv 2023-11-13
--------------------------------------------------------
[17:06:35] ... Карелия Респ, id_region: 11, Execute: /home/n.chadaev@abrr.local/Y_BUILD/10/stage_gar_c_10.csv, process_id:31561
[17:06:35] ... Start process
[17:06:35] ...  -- Version
[17:06:35] ...  -- Очистка от данных
[17:06:35] ...  -- Убираем индексы
[17:06:35] ...  -- Set UNLOGGED
[17:06:35] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_APARTMENT_TYPES_20231113_78e8cff6-619c-4b2e-806f-6c3c18c41d8c.XML
[17:06:35] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDR_OBJ_TYPES_20231113_15f2ec37-c925-4252-b49a-5473d975a16f.XML
[17:06:36] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ROOM_TYPES_20231113_32952955-cd74-40c1-bc84-06d94fe9559d.XML
[17:06:36] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OPERATION_TYPES_20231113_5151274c-91f4-4286-aaff-fc37112b660a.XML
[17:06:36] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_PARAM_TYPES_20231113_fe767969-20f3-4044-9ef6-67b4cb2881ce.XML
[17:06:36] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_HOUSE_TYPES_20231113_507d1000-4460-4d2f-a5b1-2ab5b9cd8a26.XML
[17:06:36] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDHOUSE_TYPES_20231113_d99b31fe-7b05-400a-ac92-23919e58da07.XML
[17:06:36] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OBJECT_LEVELS_20231113_b5fa7b3d-ea26-40c9-8b3a-9b986261ea66.XML
[17:06:36] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_TYPES_20231113_b8068ed2-dd18-4365-abb7-3dbd98698c35.XML
[17:06:36] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_KINDS_20231113_2f0f38c6-d598-4fd7-b325-2aa6aca29ed2.XML
[17:06:36] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/10/AS_REESTR_OBJECTS_20231113_091f92d9-2568-40ff-a2e3-221b77ceb23f.XML
[17:11:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/10/AS_ADDR_OBJ_20231113_84625895-7e77-485c-ad35-105904b8f0ea.XML
[17:11:44] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/10/AS_HOUSES_20231113_d31e7ba7-2d2c-4f1b-96bf-7021ae7bf18a.XML
[17:14:21] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/10/AS_STEADS_20231113_3d1a64c2-74a7-42eb-b8c2-0c309bd307eb.XML
[17:14:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/10/AS_APARTMENTS_20231113_6dd9ad68-c352-4b35-9105-f20bcebb2d18.XML
[17:18:16] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/10/AS_ROOMS_20231113_e7f4afd7-dd46-4da9-92b3-65b2a905c783.XML
[17:18:18] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/10/AS_CARPLACES_20231113_cf7598d6-22fb-4e76-8d50-f383d41931d1.XML
[17:18:19] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/10/AS_ADM_HIERARCHY_20231113_432f5f45-346e-4dee-8389-c3c31e8b760e.XML
[17:23:56] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/10/AS_MUN_HIERARCHY_20231113_1e670daf-afee-4422-a737-9302a2209d31.XML
[17:29:24] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/10/AS_CHANGE_HISTORY_20231113_d05c63c0-5822-4ba3-bf14-ddbf192b1780.XML
[17:42:10] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/10/AS_ADDR_OBJ_DIVISION_20231113_4135da48-c42a-4b36-8ec1-56d1d714b545.XML
[17:42:10] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/10/AS_ADDR_OBJ_PARAMS_20231113_dc3c2cd4-0b5c-4ed3-8ce7-37011e1c5ffc.XML
[17:43:56] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/10/AS_STEADS_PARAMS_20231113_b7c24fa3-45ea-4384-b871-921c38fc0e38.XML
[17:46:21] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/10/AS_HOUSES_PARAMS_20231113_a0e068f4-6e4c-44f9-b996-02b394fe2cf4.XML
[18:04:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/10/AS_APARTMENTS_PARAMS_20231113_b3b3d958-6556-4d84-a7cf-50eff2bbaf27.XML
[18:11:03] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/10/AS_ROOMS_PARAMS_20231113_29cb654e-cd3a-4d72-8a18-1dfef24fe0f6.XML
[18:11:06] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/10/AS_CARPLACES_PARAMS_20231113_d9b281ed-478a-4247-90ed-2af6bccd0a4e.XML
[18:11:06] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/10/AS_NORMATIVE_DOCS_20231113_829191c9-d559-4c7a-b5d3-fa6b66c54ccb.XML
[18:11:57] ...  -- Data finish
[18:11:57] ...  -- Set LOGGED
[18:12:07] ... Stop process
--------------------------------------------------------
