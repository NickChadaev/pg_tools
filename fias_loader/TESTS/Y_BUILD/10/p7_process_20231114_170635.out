 save_gar_version 
------------------
               85
(1 row)

CALL
CALL
CALL
 set_gar_dt_create 
-------------------
                85
(1 row)

CALL
gar_fias.as_apartment_type: 13
gar_fias.as_addr_obj_type: 421
gar_fias.as_room_type: 3
gar_fias.as_operation_type: 27
gar_fias.as_param_type: 20
gar_fias.as_house_type: 14
gar_fias.as_add_house_type: 4
gar_fias.as_object_level: 17
gar_fias.as_norm_docs_types: 25
gar_fias.as_norm_docs_kinds: 4
gar_fias.as_reestr_objects: 406063
gar_fias.as_addr_obj: 12139
gar_fias.as_houses: 191481
gar_fias.as_steads: 15589
gar_fias.as_apartments: 287654
gar_fias.as_rooms: 2932
gar_fias.as_carplaces: 685
gar_fias.as_adm_hierarchy: 431072
gar_fias.as_mun_hierarchy: 445475
gar_fias.as_change_history: 1070968
gar_fias.as_addr_obj_division: 65
gar_fias.as_addr_obj_params: 139329
gar_fias.as_steads_params: 188663
gar_fias.as_houses_params: 1421013
gar_fias.as_apartments_params: 519169
gar_fias.as_rooms_params: 3977
gar_fias.as_carplaces_params: 13
gar_fias.as_normative_docs: 66151
